test/TestLogisticRegression.cpp

```diff
@@ -162,3 +156,16 @@ TEST(TestLogisticRegression, TestMainLogisticRegressionManualCopy)
                     bIn->data()[0]);
     }
 }
+
+int
+main(int argc, char* argv[])
```

Comment:
What are the benefits of having a main on every file instead of a central one? Seems like could be code repetition that could be prone to disparities

Reply:
In my eyes a group of tests is a atomic group without or a reduced set of dependencies. But I see your point there and I can change it again if you want.

Reply:
I removed all the main methods completely since if we link against `gtest_main`, we don't need it. Regarding the initializing of the logger, this will be done inside the `Manager` when required anyway.

For reference:
https://github.com/COM8/kompute/commit/43c156ec7276489359eaa8caffcbafd7eb59f0fd

Reply:
Merged all tests into a single executable again, since ctest can not pass args to gtest.